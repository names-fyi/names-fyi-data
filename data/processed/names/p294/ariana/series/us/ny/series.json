[{"y":1966,"t":5,"m":0,"f":5,"mr":null,"fr":1402,"mp":0,"fp":0.00003429378802323747},{"y":1967,"t":9,"m":0,"f":9,"mr":null,"fr":1011,"mp":0,"fp":0.00006452167928423951},{"y":1968,"t":6,"m":0,"f":6,"mr":null,"fr":1302,"mp":0,"fp":0.00004405156970426713},{"y":1970,"t":8,"m":0,"f":8,"mr":null,"fr":1206,"mp":0,"fp":0.00005658388915216115},{"y":1971,"t":15,"m":0,"f":15,"mr":null,"fr":771,"mp":0,"fp":0.00011981883392310826},{"y":1972,"t":9,"m":0,"f":9,"mr":null,"fr":1049,"mp":0,"fp":0.0000812179076462148},{"y":1974,"t":6,"m":0,"f":6,"mr":null,"fr":1393,"mp":0,"fp":0.00005817222857807683},{"y":1975,"t":8,"m":0,"f":8,"mr":null,"fr":1157,"mp":0,"fp":0.00007858160208241245},{"y":1976,"t":11,"m":0,"f":11,"mr":null,"fr":936,"mp":0,"fp":0.00010967536093164234},{"y":1977,"t":16,"m":0,"f":16,"mr":null,"fr":746,"mp":0,"fp":0.00015713542127024347},{"y":1978,"t":18,"m":0,"f":18,"mr":null,"fr":679,"mp":0,"fp":0.00018306449972540324},{"y":1979,"t":15,"m":0,"f":15,"mr":null,"fr":781,"mp":0,"fp":0.00015164228595691338},{"y":1980,"t":27,"m":0,"f":27,"mr":null,"fr":510,"mp":0,"fp":0.0002703406291927829},{"y":1981,"t":27,"m":0,"f":27,"mr":null,"fr":515,"mp":0,"fp":0.0002664508743536099},{"y":1982,"t":33,"m":0,"f":33,"mr":null,"fr":448,"mp":0,"fp":0.0003211835125796876},{"y":1983,"t":47,"m":0,"f":47,"mr":null,"fr":338,"mp":0,"fp":0.0004569495216613518},{"y":1984,"t":35,"m":0,"f":35,"mr":null,"fr":435,"mp":0,"fp":0.0003354032505366452},{"y":1985,"t":27,"m":0,"f":27,"mr":null,"fr":540,"mp":0,"fp":0.0002526055797765844},{"y":1986,"t":51,"m":0,"f":51,"mr":null,"fr":341,"mp":0,"fp":0.0004701544134593224},{"y":1987,"t":51,"m":0,"f":51,"mr":null,"fr":352,"mp":0,"fp":0.0004559596609805815},{"y":1988,"t":68,"m":0,"f":68,"mr":null,"fr":270,"mp":0,"fp":0.000596292463915537},{"y":1989,"t":97,"m":0,"f":97,"mr":null,"fr":202,"mp":0,"fp":0.0008166357972722681},{"y":1990,"t":110,"m":0,"f":110,"mr":null,"fr":185,"mp":0,"fp":0.0009014029107119444},{"y":1991,"t":154,"m":0,"f":154,"mr":null,"fr":140,"mp":0,"fp":0.0012940524007192915},{"y":1992,"t":120,"m":0,"f":120,"mr":null,"fr":171,"mp":0,"fp":0.0010336985734959686},{"y":1993,"t":138,"m":0,"f":138,"mr":null,"fr":148,"mp":0,"fp":0.0012204612989953304},{"y":1994,"t":175,"m":0,"f":175,"mr":null,"fr":124,"mp":0,"fp":0.0015735146022155086},{"y":1995,"t":158,"m":0,"f":158,"mr":null,"fr":137,"mp":0,"fp":0.0014696990837635459},{"y":1996,"t":193,"m":0,"f":193,"mr":null,"fr":111,"mp":0,"fp":0.0018500589526557452},{"y":1997,"t":191,"m":0,"f":191,"mr":null,"fr":110,"mp":0,"fp":0.0018913512764145526},{"y":1998,"t":193,"m":0,"f":193,"mr":null,"fr":102,"mp":0,"fp":0.001916545848146015},{"y":1999,"t":198,"m":0,"f":198,"mr":null,"fr":102,"mp":0,"fp":0.001983749286150825},{"y":2000,"t":263,"m":0,"f":263,"mr":null,"fr":76,"mp":0,"fp":0.0026190523611304747},{"y":2001,"t":251,"m":0,"f":251,"mr":null,"fr":80,"mp":0,"fp":0.002562505742667252},{"y":2002,"t":290,"m":0,"f":290,"mr":null,"fr":64,"mp":0,"fp":0.002993888338288735},{"y":2003,"t":324,"m":0,"f":324,"mr":null,"fr":56,"mp":0,"fp":0.0033033247351732715},{"y":2004,"t":301,"m":0,"f":301,"mr":null,"fr":61,"mp":0,"fp":0.003103508717663192},{"y":2005,"t":312,"m":0,"f":312,"mr":null,"fr":56,"mp":0,"fp":0.003282690122469593},{"y":2006,"t":317,"m":0,"f":317,"mr":null,"fr":53,"mp":0,"fp":0.003284906012310626},{"y":2007,"t":297,"m":0,"f":297,"mr":null,"fr":51,"mp":0,"fp":0.0030392337447043656},{"y":2008,"t":288,"m":0,"f":288,"mr":null,"fr":52,"mp":0,"fp":0.0030154857759117134},{"y":2009,"t":287,"m":0,"f":287,"mr":null,"fr":53,"mp":0,"fp":0.003029183598079054},{"y":2010,"t":257,"m":0,"f":257,"mr":null,"fr":59,"mp":0,"fp":0.002762816998312209},{"y":2011,"t":244,"m":0,"f":244,"mr":null,"fr":64,"mp":0,"fp":0.002631068170544976},{"y":2012,"t":275,"m":0,"f":275,"mr":null,"fr":50,"mp":0,"fp":0.0029815469349697508},{"y":2013,"t":306,"m":0,"f":306,"mr":null,"fr":45,"mp":0,"fp":0.003354784953899115},{"y":2014,"t":398,"m":0,"f":398,"mr":null,"fr":24,"mp":0,"fp":0.004307033016979233},{"y":2015,"t":354,"m":0,"f":354,"mr":null,"fr":30,"mp":0,"fp":0.0038792820040765338},{"y":2016,"t":343,"m":0,"f":343,"mr":null,"fr":33,"mp":0,"fp":0.003820024501614879},{"y":2017,"t":298,"m":0,"f":298,"mr":null,"fr":45,"mp":0,"fp":0.003389522054641825},{"y":2018,"t":278,"m":0,"f":278,"mr":null,"fr":48,"mp":0,"fp":0.0032096798402087446},{"y":2019,"t":273,"m":0,"f":273,"mr":null,"fr":47,"mp":0,"fp":0.0032196433625813754},{"y":2020,"t":234,"m":0,"f":234,"mr":null,"fr":51,"mp":0,"fp":0.0029455081001472753},{"y":2021,"t":196,"m":0,"f":196,"mr":null,"fr":66,"mp":0,"fp":0.0024165608393850097},{"y":2022,"t":194,"m":0,"f":194,"mr":null,"fr":65,"mp":0,"fp":0.0024357783190618485}]