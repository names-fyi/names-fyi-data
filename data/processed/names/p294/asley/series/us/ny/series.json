[{"y":1986,"t":7,"m":0,"f":7,"mr":null,"fr":1392,"mp":0,"fp":0.00006453099792578935},{"y":1997,"t":5,"m":0,"f":5,"mr":null,"fr":1965,"mp":0,"fp":0.00004951181351870556}]