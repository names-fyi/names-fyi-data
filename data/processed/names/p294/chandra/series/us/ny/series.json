[{"y":1957,"t":9,"m":0,"f":9,"mr":null,"fr":897,"mp":0,"fp":0.00005312304477682419},{"y":1958,"t":16,"m":0,"f":16,"mr":null,"fr":693,"mp":0,"fp":0.00009433072351664937},{"y":1959,"t":11,"m":0,"f":11,"mr":null,"fr":853,"mp":0,"fp":0.00006535015802856397},{"y":1960,"t":7,"m":0,"f":7,"mr":null,"fr":1114,"mp":0,"fp":0.00004173572936168183},{"y":1961,"t":5,"m":0,"f":5,"mr":null,"fr":1352,"mp":0,"fp":0.00002978069496229764},{"y":1962,"t":6,"m":0,"f":6,"mr":null,"fr":1213,"mp":0,"fp":0.000036733419452794494},{"y":1963,"t":10,"m":0,"f":10,"mr":null,"fr":928,"mp":0,"fp":0.00006126024577610606},{"y":1964,"t":17,"m":0,"f":17,"mr":null,"fr":705,"mp":0,"fp":0.00010581417785495989},{"y":1965,"t":25,"m":0,"f":25,"mr":null,"fr":575,"mp":0,"fp":0.00016265769663688948},{"y":1966,"t":24,"m":0,"f":24,"mr":null,"fr":582,"mp":0,"fp":0.00016461018251153986},{"y":1967,"t":29,"m":0,"f":29,"mr":null,"fr":523,"mp":0,"fp":0.00020790318880477173},{"y":1968,"t":30,"m":0,"f":30,"mr":null,"fr":506,"mp":0,"fp":0.00022025784852133565},{"y":1969,"t":30,"m":0,"f":30,"mr":null,"fr":535,"mp":0,"fp":0.00021590500179920834},{"y":1970,"t":41,"m":0,"f":41,"mr":null,"fr":471,"mp":0,"fp":0.0002899924319048259},{"y":1971,"t":27,"m":0,"f":27,"mr":null,"fr":549,"mp":0,"fp":0.00021567390106159488},{"y":1972,"t":32,"m":0,"f":32,"mr":null,"fr":482,"mp":0,"fp":0.00028877478274209705},{"y":1973,"t":38,"m":0,"f":38,"mr":null,"fr":425,"mp":0,"fp":0.00036774668060233037},{"y":1974,"t":34,"m":0,"f":34,"mr":null,"fr":451,"mp":0,"fp":0.000329642628609102},{"y":1975,"t":28,"m":0,"f":28,"mr":null,"fr":515,"mp":0,"fp":0.00027503560728844357},{"y":1976,"t":25,"m":0,"f":25,"mr":null,"fr":545,"mp":0,"fp":0.00024926218393555075},{"y":1977,"t":30,"m":0,"f":30,"mr":null,"fr":492,"mp":0,"fp":0.0002946289148817065},{"y":1978,"t":24,"m":0,"f":24,"mr":null,"fr":570,"mp":0,"fp":0.000244085999633871},{"y":1979,"t":14,"m":0,"f":14,"mr":null,"fr":813,"mp":0,"fp":0.00014153280022645247},{"y":1980,"t":15,"m":0,"f":15,"mr":null,"fr":775,"mp":0,"fp":0.00015018923844043494},{"y":1981,"t":14,"m":0,"f":14,"mr":null,"fr":817,"mp":0,"fp":0.00013815971262779773},{"y":1982,"t":22,"m":0,"f":22,"mr":null,"fr":596,"mp":0,"fp":0.00021412234171979173},{"y":1983,"t":12,"m":0,"f":12,"mr":null,"fr":908,"mp":0,"fp":0.00011666796297736641},{"y":1984,"t":16,"m":0,"f":16,"mr":null,"fr":745,"mp":0,"fp":0.00015332720024532353},{"y":1985,"t":12,"m":0,"f":12,"mr":null,"fr":922,"mp":0,"fp":0.00011226914656737085},{"y":1986,"t":9,"m":0,"f":9,"mr":null,"fr":1180,"mp":0,"fp":0.00008296842590458631},{"y":1987,"t":12,"m":0,"f":12,"mr":null,"fr":982,"mp":0,"fp":0.000107284626113078},{"y":1988,"t":18,"m":0,"f":18,"mr":null,"fr":759,"mp":0,"fp":0.00015784212280117155},{"y":1989,"t":6,"m":0,"f":6,"mr":null,"fr":1721,"mp":0,"fp":0.00005051355447044957},{"y":1990,"t":11,"m":0,"f":11,"mr":null,"fr":1122,"mp":0,"fp":0.00009014029107119444},{"y":1991,"t":11,"m":0,"f":11,"mr":null,"fr":1144,"mp":0,"fp":0.00009243231433709224},{"y":1992,"t":8,"m":0,"f":8,"mr":null,"fr":1419,"mp":0,"fp":0.00006891323823306458},{"y":1993,"t":6,"m":0,"f":6,"mr":null,"fr":1778,"mp":0,"fp":0.00005306353473892741},{"y":1994,"t":5,"m":0,"f":5,"mr":null,"fr":2028,"mp":0,"fp":0.000044957560063300245},{"y":1999,"t":5,"m":0,"f":5,"mr":null,"fr":2062,"mp":0,"fp":0.000050094678943202656}]