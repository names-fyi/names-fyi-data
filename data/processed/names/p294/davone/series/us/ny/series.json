[{"y":1980,"t":5,"m":5,"f":0,"mr":1139,"fr":null,"mp":0.00004442312134619827,"fp":0},{"y":1988,"t":7,"m":7,"f":0,"mr":1053,"fr":null,"mp":0.00005398485339256243,"fp":0},{"y":1990,"t":7,"m":7,"f":0,"mr":1173,"fr":null,"mp":0.000050427553615295395,"fp":0},{"y":1991,"t":5,"m":5,"f":0,"mr":1496,"fr":null,"mp":0.000037202657757870225,"fp":0},{"y":1992,"t":7,"m":7,"f":0,"mr":1190,"fr":null,"mp":0.00005302026131414505,"fp":0},{"y":1995,"t":5,"m":5,"f":0,"mr":1551,"fr":null,"mp":0.00004087806074479827,"fp":0}]