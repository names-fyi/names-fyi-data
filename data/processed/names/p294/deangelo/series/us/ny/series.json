[{"y":1987,"t":5,"m":5,"f":0,"mr":1280,"fr":null,"mp":0.000039800678203556586,"fp":0},{"y":1997,"t":5,"m":5,"f":0,"mr":1565,"fr":null,"mp":0.000043527465830939325,"fp":0},{"y":1998,"t":6,"m":6,"f":0,"mr":1399,"fr":null,"mp":0.00005203229471091724,"fp":0},{"y":2002,"t":8,"m":8,"f":0,"mr":1194,"fr":null,"mp":0.00007262559689162446,"fp":0},{"y":2008,"t":5,"m":5,"f":0,"mr":1866,"fr":null,"mp":0.00004583161464778404,"fp":0},{"y":2010,"t":9,"m":9,"f":0,"mr":1271,"fr":null,"mp":0.00008473937933112383,"fp":0},{"y":2011,"t":5,"m":5,"f":0,"mr":1852,"fr":null,"mp":0.00004790052019964937,"fp":0},{"y":2012,"t":6,"m":6,"f":0,"mr":1659,"fr":null,"mp":0.000057545124968829724,"fp":0},{"y":2014,"t":6,"m":6,"f":0,"mr":1637,"fr":null,"mp":0.000058428848270019186,"fp":0},{"y":2017,"t":6,"m":6,"f":0,"mr":1610,"fr":null,"mp":0.00006155108740254411,"fp":0},{"y":2018,"t":5,"m":5,"f":0,"mr":1836,"fr":null,"mp":0.000052281567610523236,"fp":0},{"y":2020,"t":5,"m":5,"f":0,"mr":1768,"fr":null,"mp":0.00005715918833952558,"fp":0}]