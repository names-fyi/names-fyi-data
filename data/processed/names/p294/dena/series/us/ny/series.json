[{"y":1916,"t":5,"m":0,"f":5,"mr":null,"fr":677,"mp":0,"fp":0.00005320563979781857},{"y":1918,"t":6,"m":0,"f":6,"mr":null,"fr":643,"mp":0,"fp":0.00005931256734447751},{"y":1920,"t":7,"m":0,"f":7,"mr":null,"fr":581,"mp":0,"fp":0.00007087889833940866},{"y":1923,"t":5,"m":0,"f":5,"mr":null,"fr":698,"mp":0,"fp":0.00005018014672674903},{"y":1927,"t":8,"m":0,"f":8,"mr":null,"fr":594,"mp":0,"fp":0.00007936744149131422},{"y":1932,"t":5,"m":0,"f":5,"mr":null,"fr":699,"mp":0,"fp":0.00005675626589175445},{"y":1938,"t":5,"m":0,"f":5,"mr":null,"fr":709,"mp":0,"fp":0.00005844398728258837},{"y":1939,"t":5,"m":0,"f":5,"mr":null,"fr":717,"mp":0,"fp":0.00005869302374719741},{"y":1941,"t":9,"m":0,"f":9,"mr":null,"fr":571,"mp":0,"fp":0.0000933116297394532},{"y":1942,"t":13,"m":0,"f":13,"mr":null,"fr":509,"mp":0,"fp":0.00011499438296667816},{"y":1943,"t":12,"m":0,"f":12,"mr":null,"fr":547,"mp":0,"fp":0.0001048474469646664},{"y":1944,"t":12,"m":0,"f":12,"mr":null,"fr":516,"mp":0,"fp":0.00011377858686995108},{"y":1945,"t":9,"m":0,"f":9,"mr":null,"fr":638,"mp":0,"fp":0.00008310479514667997},{"y":1946,"t":11,"m":0,"f":11,"mr":null,"fr":632,"mp":0,"fp":0.00008300445960323868},{"y":1947,"t":24,"m":0,"f":24,"mr":null,"fr":453,"mp":0,"fp":0.00015956067627133293},{"y":1948,"t":9,"m":0,"f":9,"mr":null,"fr":742,"mp":0,"fp":0.00006366723259762309},{"y":1949,"t":13,"m":0,"f":13,"mr":null,"fr":630,"mp":0,"fp":0.00009161574945206735},{"y":1950,"t":14,"m":0,"f":14,"mr":null,"fr":614,"mp":0,"fp":0.00009802411393202727},{"y":1951,"t":14,"m":0,"f":14,"mr":null,"fr":627,"mp":0,"fp":0.00009351100424139198},{"y":1952,"t":9,"m":0,"f":9,"mr":null,"fr":804,"mp":0,"fp":0.000058276944993039145},{"y":1953,"t":17,"m":0,"f":17,"mr":null,"fr":585,"mp":0,"fp":0.00011051519583942792},{"y":1954,"t":28,"m":0,"f":28,"mr":null,"fr":479,"mp":0,"fp":0.00017455923792424128},{"y":1955,"t":27,"m":0,"f":27,"mr":null,"fr":493,"mp":0,"fp":0.00016527104451300131},{"y":1956,"t":17,"m":0,"f":17,"mr":null,"fr":648,"mp":0,"fp":0.00010318539380402058},{"y":1957,"t":22,"m":0,"f":22,"mr":null,"fr":567,"mp":0,"fp":0.00012985633167668134},{"y":1958,"t":21,"m":0,"f":21,"mr":null,"fr":598,"mp":0,"fp":0.0001238090746156023},{"y":1959,"t":31,"m":0,"f":31,"mr":null,"fr":506,"mp":0,"fp":0.00018416862717140752},{"y":1960,"t":28,"m":0,"f":28,"mr":null,"fr":529,"mp":0,"fp":0.00016694291744672731},{"y":1961,"t":23,"m":0,"f":23,"mr":null,"fr":601,"mp":0,"fp":0.00013699119682656914},{"y":1962,"t":33,"m":0,"f":33,"mr":null,"fr":497,"mp":0,"fp":0.00020203380699036972},{"y":1963,"t":34,"m":0,"f":34,"mr":null,"fr":498,"mp":0,"fp":0.0002082848356387606},{"y":1964,"t":43,"m":0,"f":43,"mr":null,"fr":434,"mp":0,"fp":0.00026764762633901617},{"y":1965,"t":47,"m":0,"f":47,"mr":null,"fr":406,"mp":0,"fp":0.0003057964696773522},{"y":1966,"t":46,"m":0,"f":46,"mr":null,"fr":399,"mp":0,"fp":0.0003155028498137847},{"y":1967,"t":43,"m":0,"f":43,"mr":null,"fr":419,"mp":0,"fp":0.0003082702454691443},{"y":1968,"t":59,"m":0,"f":59,"mr":null,"fr":365,"mp":0,"fp":0.00043317376875862677},{"y":1969,"t":58,"m":0,"f":58,"mr":null,"fr":360,"mp":0,"fp":0.00041741633681180283},{"y":1970,"t":83,"m":0,"f":83,"mr":null,"fr":295,"mp":0,"fp":0.000587057849953672},{"y":1971,"t":66,"m":0,"f":66,"mr":null,"fr":326,"mp":0,"fp":0.0005272028692616764},{"y":1972,"t":51,"m":0,"f":51,"mr":null,"fr":360,"mp":0,"fp":0.0004602348099952172},{"y":1973,"t":39,"m":0,"f":39,"mr":null,"fr":418,"mp":0,"fp":0.0003774242248287075},{"y":1974,"t":47,"m":0,"f":47,"mr":null,"fr":361,"mp":0,"fp":0.00045568245719493513},{"y":1975,"t":56,"m":0,"f":56,"mr":null,"fr":317,"mp":0,"fp":0.0005500712145768871},{"y":1976,"t":47,"m":0,"f":47,"mr":null,"fr":354,"mp":0,"fp":0.00046861290579883544},{"y":1977,"t":49,"m":0,"f":49,"mr":null,"fr":350,"mp":0,"fp":0.0004812272276401206},{"y":1978,"t":34,"m":0,"f":34,"mr":null,"fr":433,"mp":0,"fp":0.00034578849948131723},{"y":1979,"t":39,"m":0,"f":39,"mr":null,"fr":399,"mp":0,"fp":0.00039426994348797475},{"y":1980,"t":42,"m":0,"f":42,"mr":null,"fr":376,"mp":0,"fp":0.0004205298676332179},{"y":1981,"t":46,"m":0,"f":46,"mr":null,"fr":342,"mp":0,"fp":0.0004539533414913354},{"y":1982,"t":35,"m":0,"f":35,"mr":null,"fr":432,"mp":0,"fp":0.0003406491800087595},{"y":1983,"t":45,"m":0,"f":45,"mr":null,"fr":349,"mp":0,"fp":0.00043750486116512404},{"y":1984,"t":31,"m":0,"f":31,"mr":null,"fr":478,"mp":0,"fp":0.0002970714504753143},{"y":1985,"t":25,"m":0,"f":25,"mr":null,"fr":570,"mp":0,"fp":0.00023389405534868926},{"y":1986,"t":20,"m":0,"f":20,"mr":null,"fr":649,"mp":0,"fp":0.00018437427978796957},{"y":1987,"t":17,"m":0,"f":17,"mr":null,"fr":766,"mp":0,"fp":0.00015198655366019381},{"y":1988,"t":27,"m":0,"f":27,"mr":null,"fr":579,"mp":0,"fp":0.0002367631842017573},{"y":1989,"t":21,"m":0,"f":21,"mr":null,"fr":706,"mp":0,"fp":0.0001767974406465735},{"y":1990,"t":16,"m":0,"f":16,"mr":null,"fr":889,"mp":0,"fp":0.0001311131506490101},{"y":1991,"t":19,"m":0,"f":19,"mr":null,"fr":785,"mp":0,"fp":0.00015965581567315933},{"y":1992,"t":7,"m":0,"f":7,"mr":null,"fr":1582,"mp":0,"fp":0.0000602990834539315},{"y":1993,"t":16,"m":0,"f":16,"mr":null,"fr":884,"mp":0,"fp":0.00014150275930380644},{"y":1994,"t":8,"m":0,"f":8,"mr":null,"fr":1464,"mp":0,"fp":0.00007193209610128039},{"y":1995,"t":5,"m":0,"f":5,"mr":null,"fr":2050,"mp":0,"fp":0.00004650946467606158},{"y":1996,"t":18,"m":0,"f":18,"mr":null,"fr":774,"mp":0,"fp":0.00017254435827877417},{"y":1997,"t":11,"m":0,"f":11,"mr":null,"fr":1126,"mp":0,"fp":0.00010892598974115224},{"y":1998,"t":7,"m":0,"f":7,"mr":null,"fr":1601,"mp":0,"fp":0.00006951202558042541},{"y":1999,"t":15,"m":0,"f":15,"mr":null,"fr":922,"mp":0,"fp":0.00015028403682960795},{"y":2000,"t":6,"m":0,"f":6,"mr":null,"fr":1867,"mp":0,"fp":0.00005975024398016292},{"y":2001,"t":7,"m":0,"f":7,"mr":null,"fr":1698,"mp":0,"fp":0.0000714643035803616},{"y":2003,"t":7,"m":0,"f":7,"mr":null,"fr":1691,"mp":0,"fp":0.00007136812699448426},{"y":2004,"t":5,"m":0,"f":5,"mr":null,"fr":2207,"mp":0,"fp":0.00005155330095786033},{"y":2005,"t":8,"m":0,"f":8,"mr":null,"fr":1561,"mp":0,"fp":0.00008417154160178443},{"y":2006,"t":8,"m":0,"f":8,"mr":null,"fr":1637,"mp":0,"fp":0.00008289983627282336},{"y":2007,"t":9,"m":0,"f":9,"mr":null,"fr":1492,"mp":0,"fp":0.00009209799226376865},{"y":2008,"t":5,"m":0,"f":5,"mr":null,"fr":2340,"mp":0,"fp":0.00005235218360957836},{"y":2009,"t":9,"m":0,"f":9,"mr":null,"fr":1505,"mp":0,"fp":0.00009499182014882052},{"y":2014,"t":6,"m":0,"f":6,"mr":null,"fr":1994,"mp":0,"fp":0.00006493014598461156},{"y":2018,"t":6,"m":0,"f":6,"mr":null,"fr":1984,"mp":0,"fp":0.00006927366561601607}]