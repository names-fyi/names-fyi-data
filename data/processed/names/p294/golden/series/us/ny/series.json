[{"y":2020,"t":7,"m":7,"f":0,"mr":1444,"fr":null,"mp":0.00008002286367533581,"fp":0}]